src/server/verification/verificationAPI.js

```diff
@@ -39,69 +35,41 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
   app.post(
     '/verify/facerecognition',
     passport.authenticate('jwt', { session: false }),
-    upload.any(),
     wrapAsync(async (req, res, next) => {
+      const { sessionId, images } = req.body
+      const { user } = req
       const log = req.log
-      const { body, files, user } = req
-      log.debug({ user })
-      const sessionId = body.sessionId
+      let result = { ok: 0 }
+
+      if (user.identifier === undefined || sessionId === undefined || images === undefined) {
+        return res.status(400).send({ ok: 0, error: 'invalid input', isVerified: false })
+      }
       GunDBPublic.gun
         .get(sessionId)
         .get('isStarted')
-        .put(true) // publish initialized data to subscribers
+        .put(true)
       log.debug('written FR status to gun', { data: await GunDBPublic.gun.get(sessionId) })
 
-      const verificationData = {
-        facemapFile: _.get(_.find(files, { fieldname: 'facemap' }), 'path', ''),
-        auditTrailImageFile: _.get(_.find(files, { fieldname: 'auditTrailImage' }), 'path', ''),
-        enrollmentIdentifier: body.enrollmentIdentifier,
-        sessionId: sessionId
-      }
-      let result = { ok: 0 }
-      if (!user.isVerified && !conf.skipFaceRecognition)
-        result = await verifier
-          .verifyUser(user, verificationData)
-          .catch(e => {
-            log.error('Facerecognition error:', { e })
-            GunDBPublic.gun.get(sessionId).put({ isNotDuplicate: false, isLive: false, isError: e.message })
-            return { ok: 1, error: e.message, isVerified: false }
-          })
-          .finally(() => {
-            //cleanup
-            log.info('cleaning up facerecognition files')
-            fsPromises.unlink(verificationData.facemapFile)
-            fsPromises.unlink(verificationData.auditTrailImageFile)
-          })
-      else {
+      const imagesBase64 = images.map(img => img.base64)
+
+      if (!user.isVerified && !conf.skipFaceRecognition) {
+        try {
+          result = await Verifications.verifyUser(user, sessionId, imagesBase64, storage)
+        } catch (e) {
+          log.error('Facerecognition error:', { e })
+          GunDBPublic.gun.get(sessionId).put({ isNotDuplicate: false, isLive: false, isError: e.message })
```

Comment:
modify to match new backend field names

Reply:
done in https://github.com/GoodDollar/GoodServer/pull/151/commits/46d0a0ef6b5ae651fa64ca55c721364da337d9d9